Yoast/javascript
# Add passive voice assessment for Hebrew

## Summary


This PR can be summarized in the following changelog entry:

* [yoastseo] Adds passive voice assessment for Hebrew, including a check, a number or specs and a list of Hebrew verb roots, both in their original and (for some) modified version.
* [Yoast SEO Free] Adds an assessment for the recognition of the passive voice in Hebrew texts (for example: `.העיר הזו בורכה במזל טוב אחרי כישוף ארוך ומייבש` or `.הוא הומלך בגיל 14 ושלט 50 שנה`)

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

- Set the environment language to Hebrew (for team lingo: locale `he_IL`)
- Test this functionality in Wordpress SEO Free (for team lingo: morphology `off`)
- You should see a bullet assessment for passive verb forms in the Readability section
- Add these sentences to your post:
`.מעל גופתו של קדוש מבורך, הוא הומלך למלך`
`!כאשר שטן גורש מ שמיים, הוא קולל לזחול כדור הארץ כמו נחש`
`.וכתביו בנושאי פילוסופיה ייקראו במשך דורות`
- Check that the passive voice bullet turns red

Extra for internal testing: 
- run `yarn test` and make sure all tests pass

## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  *

## UI changes

## Quality assurance


Fixes https://yoast.atlassian.net/browse/LIN-517
